examples/soak/bin/run_task.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
what's this doing? assuming soak's deployed as a service, not a task?

Reply:
It is - should kill this, I think we were considering keeping it to manually trigger the ingest task though if we do want a script to run that we should include it in that PR